Updated doc on how to enable Symfony Reverse Proxy (#2196)
ibexa/documentation-developer: docs/guide/cache/symfony_reverse_proxy.md

# Reverse proxy

## Using Symfony reverse proxy

To use the Symfony reverse proxy, you must change your `public/index.php` front controller script and wrap `EzSystems\PlatformHttpCacheBundle\AppCache` instead of `Symfony\Bundle\FrameworkBundle\HttpCache\HttpCache` around the kernel.

```diff
--- a/public/index.php
+++ b/public/index.php
@@ -1,9 +1,11 @@
 <?php

 use App\Kernel;
+use EzSystems\PlatformHttpCacheBundle\AppCache;

 require_once dirname(__DIR__).'/vendor/autoload_runtime.php';

 return function (array $context) {
-    return new Kernel($context['APP_ENV'], (bool) $context['APP_DEBUG']);
+    return new AppCache(new Kernel($context['APP_ENV'], (bool) $context['APP_DEBUG']));
 };
```

!!! caution

>     Do not enable the Symfony reverse proxy in `public/index.php` if you intend to use Varnish or Fastly.

Reviewer: ```suggestion
    Don't enable the Symfony reverse proxy in `public/index.php` if you intend to use Varnish or Fastly.
```
